package.json

```diff
@@ -104,7 +105,7 @@
     "parse-svg-path": "^0.1.2",
     "polybooljs": "^1.2.0",
     "probe-image-size": "^7.2.2",
-    "regl": "^2.1.0",
+    "regl": "/mnt/c/Users/yujin/plotly/regl",
```

Comment:
Let's link to your regl PR here
```js
"regl": "git@github.com:yujin-wu/regl.git#c26cb99bf8dc52b37e6f2040be8cd3d82967fc9f",
```
then `npm install` and commit `package-lock.json` diff as well.

Reply:
Done, it is testable now

Reply:
I am still wrestling with CI here but locally it will work fine